demo/site/src/app/[[...path]]/page.tsx

```diff
@@ -28,10 +33,49 @@ export default async function Page({ params }: { params: { path: string[] } }) {
     const scope = { domain, language: locale };
 
     //fetch documentType
-    const data = await graphqlFetch<GQLDocumentTypeQuery, GQLDocumentTypeQueryVariables>(documentTypeQuery, {
-        path: `/${(params.path ?? []).join("/")}`,
+    const data = await graphqlFetch<GQLDocumentTypeQuery, GQLDocumentTypeQueryVariables>(
+        documentTypeQuery,
+        {
+            path: `/${(params.path ?? []).join("/")}`,
+            scope,
+        },
+        { method: "GET" }, //for request memoization
+    );
```

Comment:
Should we move this into a `fetchPageTreeNode` function and reuse it?

Reply:
done in 9e7d81c4bf67983f5d493406f40974a3f22385ae

I'm not sure if it's really a good idea as it requires 3 arguments

Reply:
refactored again in 7f6d026b23cd7d922602e1bfb6435a8e7f561c7c

still not ideal, scope is also needed outside

Reply:
> I'm not sure if it's really a good idea as it requires 3 arguments

You can also revert. I'd rather have duplicate code than a hasty abstraction. Haven't thought it through when making the suggestion.